draft-ietf-quic-multipath.md

```diff
@@ -110,19 +110,27 @@ This document specifies the needed path management mechanisms for path
 initiation {{path-initiation}}, handling of per-path connection IDs {{consume-retire-cid}},
 signaling of preferred path usage {{path-state}}, and explicit
 removal of paths that have been abandoned {{path-close}}.
-However, this document does not specify detailed algorithms that define
+Note that in this extention, a QUIC server does not initiate the creation
+of a path, but it has to validate a new path created by a client.
+
+This proposal does not cover address discovery and management. Addresses
+and the actual decision to setup or tear down paths are assumed
+to be handled by the application. But this document does not prevent future extensions from
+defining mechanisms to address the remaining scenarios.
```

Comment:
Perhaps avoid using `address` here since it means something different to the earlier address discovery mention